### ISSUE TITLE
Chromatically alter tone in a given layer.

### ISSUE THREAD

**Issue by yrammos** [DCMLab/reductive_analysis_app#103]
Bumps one of several issues already filed in #29.

**Comment by pettter** [comment]
I've been thinking a little bit about how to achieve this, and unfortunately haven't really come up with a satisfying answer. Verovio doesn't seem to include unused glyphs in their SVG preambles, so we can't depend on Verovio including the accidentals _for_ us. 

_Deleting_ or _hiding_ accidentals, is much simpler, generally, and may be enough for the common cases. I'll start there.

**Comment by yrammos** [comment]
@yinanazhou removing accidentals is implemented (in bare-bones fashion) in #127 (see `js/naturalize_notes.js`). There is no graphic interface; the feature relies on a shortcut (by default uppercase `Z`, declared in `conf.js`).

Adding accidentals is not supported at all at the moment.

**Comment by yinanazhou** [comment]
@yrammos yes, I saw that removing accidentals is implemented but adding accidentals is more complicated because we need verovio to re-render SVG.
